src/esm_runscripts/compute.py

```diff
@@ -45,9 +45,9 @@ def compile_model(config):
     if not version:
         return config
     if config.get("general", {}).get("run_number") == 1:
-        print("First year, checking if we need to compile...")
+        logger.info("First year, checking if we need to compile...")
         if not config.get("general", {}).get("use_compiled_model", True):
-            print(f"Huhu --> compiling {model}-{version}")
+            logger.debug(f"Huhu --> compiling {model}-{version}")
```

Comment:
```suggestion
            logger.info(f"Huhu --> compiling {model}-{version}")
```
I think when you inform about the checking whether to compile or not you can then also inform about when it is compiling.

Reply:
Is it correct, that `logger.info` will always be written to log file and `logger.debug` only in debug mode?

Reply:
Yes, `logger.info` is what used to be be `print`, and `logger.debug` is what used to be print in verbose mode